lms/templates/courseware/course_navigation-sneak-peek-warning.html

```diff
@@ -13,11 +13,15 @@
     reg_url = reverse('course-specific-register', args=[course.id.to_deprecated_string()])
   else:
     reg_url = reverse('register_user')
+  tos_link = u"<a href='{}'>".format(marketing_link('TOS'))
+  privacy_link = u"<a href='{}#privacy'>".format(marketing_link('TOS'))
   %>
-   ${_("Non-registered mode.  {tag_start}Register{tag_end} to save your course progress.")\
+   ${_("Non-registered mode.  {tag_start}Register{tag_end} to save your course progress. By exploring the course, you are agreeing to our {tos_link_start}Terms of Service{tag_end} and {privacy_link_start}Privacy Policy{tag_end}. Please read them carefully.")\
```

Comment:
This is really long now. What happens when the window is smaller like on a laptop or tablet?

I suspect we need to trim some words, or at the very least style for multiple lines.